lua/dingllm.lua

```diff
@@ -115,8 +110,7 @@ local function get_prompt(opts)
   if visual_lines then
     prompt = table.concat(visual_lines, '\n')
     if replace then
-      vim.api.nvim_command 'normal! d'
-      vim.api.nvim_command 'normal! k'
+      vim.api.nvim_command 'normal! c'
```

Comment:
fix problem with visual line cursor jumping to lower line after replacing
change vim.api.nvim_command 'normal! c'
to        vim.api.nvim_command 'normal! c '

Reply:
> fix problem with visual line cursor jumping to lower line after deleting change vim.api.nvim_command 'normal! c' to vim.api.nvim_command 'normal! c '